src/redux/actions/authActions/successSignInAction.ts

```diff
@@ -0,0 +1,20 @@
+import { AuthActionsTypes } from './index';
+import { FirebaseAuthTypes } from '@react-native-firebase/auth';
+import { AuthReducerState } from '../../reducers/authReducer';
+
+export const successSignInAction = (payload: SignInPayload) => {
+  return {
+    type: AuthActionsTypes.SIGN_IN,
+    payload,
+  };
+};
+
+export type SignInPayload = {
+  userToken: AuthReducerState['userToken'];
+  user: AuthReducerState['user'];
+};
+
+export type SignInAction = {
```

Comment:
action name `successSignInAction`
type for it SignInAction and type for payload SignInPayload 
why? we already discussed name styling for actions

Reply:
i fix it